docs/api/cloudlink/packets/ulist.md

```diff
@@ -0,0 +1,31 @@
+# `ulist`
+
+This packet provides a string of online users separated by ";"
+## Payload
+
+A string of users separated by ";"
+```
+user1;user2;user3;
+```
+
+## Usage
+::: code-group
+
+```ts [TypeScript]
+// Converts the online userlist into a array of strings
+// val is the string of users given by the websocket
+let onlineUsers: string[] = val.split(";");
```

Comment:
The JS code works in Typescript as well and is more idiomatic

Reply:
Ok, I will merge them into one.